### PR TITLE
Xdr integration

### PR DESCRIPTION
**Pull request by anara123**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/17397

## Description
XDR Integration

4 commands
1. fetch-incidents
2. list-incidents
3. get-incident-details
4. incident-update

## Does it break backward compatibility?

   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Scripts/XDRSyncScript/XDRSyncScript.yml [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+- contextPath: PaloAltoNetworks.Incident.file_artifacts.file_sha256
+  description: SHA256 of the file
+  type: String
+script: '-'
+system: false
+tags:
+- xdr
+timeout: '0'
+type: python
+dockerimage: demisto/python3:3.7.3.221
```
Comment (glicht) [inline Scripts/XDRSyncScript/XDRSyncScript.yml]:
```suggestion
dockerimage: demisto/python3:3.7.4.977
```

**Comment by anara123** [comment]
@kirbles19 please review and update
https://github.com/demisto/content/blob/xdr-integration/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.yml

**Comment by glicht** [comment]
Build is failing. Also see my remarks on the issue.

**Comment by anara123** [comment]
The build fails on all the PRs. Was something related to AMIs. Will rebuild

**Comment by glicht** [comment]
@kirbles19 please review:

* YML File of Integration: https://github.com/demisto/content/edit/xdr-integration/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F3875
* YML File of Script: https://github.com/demisto/content/edit/xdr-integration/Scripts/XDRSyncScript/XDRSyncScript.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F3875
* Script release notes: https://github.com/demisto/content/edit/xdr-integration/Releases/LatestRelease/XDRSyncScript.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F3875
* Integration Release notes: https://github.com/demisto/content/edit/xdr-integration/Releases/LatestRelease/PaloAltoNetworks_XDR.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F3875

**Comment by kirbles19** [comment]
Updated the .yml.

**Pull request merged**